packages/fxa-settings/src/components/PageTwoStepAuthentication/en-US.ftl

```diff
@@ -22,8 +22,10 @@ tfa-scan-this-code = Scan this QR code using one of <linkExternal>these
 # This is the image alt text for a QR code.
 # Variables:
 #   $secret (String) - a long alphanumeric string that does not require translation
-tfa-qa-code-alt = Use the code { $secret } to set up two-step authentication in
-  supported applications.
+# DEV NOTE: Set image alt text per fluent/react documentation, do not use the below as an example
+tfa-qa-code-alt = Use the code { $secret } to set up two-step authentication in supported applications.
+tfa-qa-code =
+  .alt = { tfa-qa-code-alt }
 tfa-button-cant-scan-qr = Can’t scan code?
```

Comment:
While the ftl ID `tfa-qa-code-alt` was translated, it was never included in the component and, therefore, was not utilized. We have revised the component within this PR to correctly localize the image alt text based on [fluent/react documentation](https://github.com/projectfluent/fluent.js/wiki/Localized). Can you please let us know if this is the best way to make this revision for the ftl ID? @peiying2
cc: @LZoog

Reply:
@peiying2 let me know in Slack or otherwise if we need to explain this further. I asked @xlisachan to ask you about this one mostly for future reference on what would be the least amount of work for us and for translators - I was wondering if changing this:

```
tfa-qa-code-alt = Use the code { $secret } to set up two-step authentication in
  supported applications.
```

... to this:
```
tfa-qa-code-alt = 
  .alt = Use the code { $secret } to set up two-step authentication in supported applications.
```
was possible, or any other alternative. I'm guessing it's not since we're changing what's tied to the ID, and what Lisa changed it to works, we're just asking because I want to document FxA + Fluent best approaches.